profiles/sftp_storage/init.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+pip install pulpcore[sftp]
```

Comment:
The base profile installs pulp & plugins from their source directory before this script is ran. I think this would re-install pulpcore from PyPI.

Reply:
> 

Thank you! I know about the issue and event contacted @newswangerd about it, but I will make sure to try this suggested solution.

Reply:
Gerrod's suggestion makes sense to me. I think that's the best way to handle this at the moment.